app/controllers/api/v1/apps_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class AppsController < ApplicationController
+      def index
+        apps = App.where('title like :prefix', prefix: "#{params[:title]}%")
+        render json: AppIndexSerializer.new(apps)
+      end
+
+      def show
+        options = {}
+        options[:include] = %i[ratings dynamic_infos]
+        app = App.find_by title: params[:title]
```

Comment:
```suggestion
        app = App.find(params[:id])
```
используйте id в запросах